cocos/core/math/quat.ts

```diff
@@ -239,6 +253,7 @@ export class Quat extends ValueType {
     }
 
     /**
+     * @en Local space around the specified axis rotation quaternion
```

Comment:
Sets the out quaternion to represent a radian rotation around a given rotation axis in local space